source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -372,20 +391,24 @@ def get_coords(param):
         """
         return param.split(";")
 
-    def get_by_uri(self, uri, request_id, instances=None, current_datetime=None):
+    def extend_params(self, params, api):
+        if api == 'multi-reverse':
+            params.append(("reverse_timeout", int(self.fast_reverse_timeout_bragi_es * 1000)))
+            params.append(("within_timeout", int(self.fast_within_timeout_bragi_es * 1000)))
+        else:
+            params.append(("timeout", int(self.fast_timeout_bragi_es * 1000)))
 
+    def get_by_uri(self, uri, request_id, instances=None, current_datetime=None):
+        api = get_api_name(uri, instances)
         params = self.basic_params(instances)
         lon, lat = get_lon_lat_from_id(uri)
```

Comment:
Done

Reply:
Well the sense of my remark was to avoid to do twice the same thing or to forward twice the same information. And you still need to forward the `api` twice (one into `make_url`, and another one as the third returned value in the tuple). Why don't you also move the following line inside the function (and I believe you won't need to return the third tuple element anymore).
```python
        self.extend_params(params, api)
```